src/app/store/api.js

```diff
@@ -3,10 +3,12 @@ import { createClient } from 'contentful-management'
 let client = null
 let space = null
 
-export async function initClient (accessToken, spaceId) {
+export async function initClient (accessToken, spaceId, host, uploadHost) {
   try {
     client = createClient({
-      accessToken
+      accessToken,
+      defaultHostname: host,
+      defaultHostnameUpload: uploadHost
```

Comment:
I guess thats a confusion we also have internally. AFAIK it works to set `host` or `defaultHostname`. But IMHO, we should encourage the user to set the `host` since it also covers the port etc.